test/test.js

```diff
@@ -1765,3 +1765,30 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test( "destruction of Validator", function() {
+	expect( 2 );
+
+	var form = $( "#validatorDestroy" ),
+		validate = form.validate();
+
+	form.on( "submit.validate", function( event )
+	{
```

Comment:
PS: Updating `grunt-jscs` to latest (1.5.0) ends up with several thousand errors :/